alica_engine/src/engine/BasicBehaviour.cpp

```diff
@@ -185,4 +196,15 @@ bool BasicBehaviour::getParameter(const std::string& key, std::string& valueOut)
     }
 }
 
+void BasicBehaviour::startTrace()
```

Comment:
Add both tags and logs in setSuccess and setFailure methods.
Ex: - 
setTag("Result", "Success" or "Fail");
setLog("Success" or "Fail");